www/js/app.js

```diff
@@ -1144,6 +1181,10 @@ define(['jquery', 'zimArchiveLoader', 'uiUtil', 'settingsStore','abstractFilesys
                     docBody.addEventListener('dragover', handleIframeDragover);
                     docBody.addEventListener('drop', handleIframeDrop);
                 }
+                // Configure home key press to focus #prefix only if the feature is in active state
+                if (docBody && params.useHomeKeyToFocusSearchBar)
+                    docBody.addEventListener('keydown', focusPrefixOnHomeKey);
```

Comment:
@Hrishi-Raj Here is my suggestion:

Instead of `docBody` here, you can use `iframeArticleContent.contentWindow.addEventListener( ..... );` This works much better with IE11.

But then you can move this code to be inside the test on line 1190 (add the missing curly braces):

```
if (iframeArticleContent.contentWindow) {
    if (params.useHomeKeyToFocusSearchBar)
        ....
    iframeArticleContent.contentWindow.onunload = function () { 
       ....
```
You can add the `.removeEventListener` code inside this `onunload` function, before the next statements.

Then you need to copy the same idea for jQuery mode and put it round about [line 1378](https://github.com/kiwix/kiwix-js/pull/714/files#diff-484c9e8aa09e09b8eb1fafaf55e8eadcd0eaab230e38d02bb9440841bfa090a4R1378).

I hope this is helpful.

Reply:
This was very helpful. I made the changes and also tested it on IE11, it works as you described (without clicking anywhere).